pysmt/solvers/msat.py

```diff
@@ -1150,3 +1157,69 @@ def sequence_interpolant(self, formulas):
                 mathsat.msat_destroy_config(cfg)
             if env:
                 mathsat.msat_destroy_env(env)
+
+
+class MSatBoolUFRewriter(IdentityDagWalker):
```

Comment:
Why is this class here? There is nothing that is mathsat-specific hence I think it could go in the rewritings.py module to be used in other situations as well (e.g. in the SMT-LIB wrapper)